proposals/tsc-voting-procedures.md

```diff
@@ -0,0 +1,199 @@
+- **Definition**: Time-sensitive matters requiring faster resolution
+- **Process**: 
+  - 3 calendar days total
+  - Requires explicit "URGENT" label on PR
+  - Must document urgency rationale
+
+### Lazy Consensus
+- **When Used**: Minor procedural changes, documentation updates
+- **Process**: 
+  - 3 calendar days review period
+  - Assumed approval if no objections
+  - Any TSC member can request full vote
+
+### Tied Votes
+- **Resolution**: 
+  - TSC Chair breaks tie
+  - If Chair abstained/absent: Defer to next meeting
+  - If still tied: Proposal fails
+
+## Templates
+
+### Proposal Template
+```markdown
+# [Proposal Title]
+
+**Date**: YYYY-MM-DD
+**Author**: [Name] (@github-username)
+**Type**: [Standard | Urgent | Charter-Required]
+
+## Summary
+Brief description of the proposal
+
+## Motivation
+Why this decision is needed
+
+## Proposal Details
+Detailed description of what is being proposed
+
+## Impact
+- Technical impact
+- Community impact
+- Timeline considerations
+
+## Alternatives Considered
+Other options that were evaluated
+
+## Vote Threshold
+- [ ] Standard (>50% of eligible voting members)
+- [ ] Charter-specified (define requirement)
+```
+
+### Voting Record Template
+```markdown
+## Voting Record
+
+**Voting Period**: YYYY-MM-DD to YYYY-MM-DD
+**Eligible Voters**: X
+**Votes Required**: Y
+
+### Results
+- **Yes**: X votes
+  - @member1
+  - @member2
+- **No**: Y votes
+  - @member3
+- **Abstain**: Z members
+  - @member4
+
+**Decision**: [Passed | Failed]
+```
+
+### Example: tsc-voting-eligibility.md
+```markdown
+# TSC Voting Eligibility
+
+Last Updated: YYYY-MM-DD
+
+## Active Voting Members
+
+| Member | GitHub Username | Status | Last Meeting Attended |
+|--------|----------------|--------|----------------------|
+| Jane Doe | @janedoe | Active | 2024-01-15 |
+| John Smith | @jsmith | Active | 2024-01-15 |
+| Alice Johnson | @alicej | Active | 2024-01-08 |
+| Bob Wilson | @bwilson | Suspended | 2023-12-01 |
+
+## Suspension Notes
+- Bob Wilson: Suspended due to missing 3 consecutive meetings (2023-12-15, 2024-01-08, 2024-01-15)
+
+## Eligibility Rules
+- Missing 3 consecutive TSC meetings results in automatic voting suspension
+- Reinstatement occurs upon attendance at next TSC meeting
+```
+
+
+## Appendix: Rationale
+
+This proposal addresses several key challenges:
+
+1. **Quorum Issues**: Asynchronous voting allows participation regardless of meeting attendance
+2. **Transparency**: GitHub PRs provide public record of all decisions
+3. **Efficiency**: Clear timelines prevent decision delays
+4. **Flexibility**: Multiple voting methods accommodate different decision types
+5. **Compliance**: Maintains charter requirements while improving process
+
+## Proposed Charter Modifications
+
+To implement this proposal effectively, the following charter modification is recommended:
+
+### Meeting Attendance Requirement
+Modify section 3(ix)(d)(d) to reduce reinstatement requirement from two meetings to one:
```

Comment:
Is this a concrete suggestion to change the charter?

Reply:
This was a suggestion from @popowycz: https://github.com/hiero-ledger/tsc/issues/159#issuecomment-2984450415 

If we feel it doesn't belong in this document, we can remove it